src/Validation/Validation.php

```diff
@@ -1431,6 +1431,17 @@ public static function isArray($value)
     }
 
     /**
+     * Check that the input value is a 6 digits hex color.
+     *
+     * @param string|array $check The value to check
+     * @return bool Success
+     */
+    public static function hexColor($check)
```

Comment:
If you only support the ones with 6 digits, I would reflect that in the method name(s).